src/sentry/models/__init__.py

```diff
@@ -1,3 +1,7 @@
+from ..users.models.auth.authenticator import *  # NOQA
+from ..users.models.users.userip import *  # NOQA
+from ..users.models.users.userreport import *  # NOQA
+from ..users.models.users.userrole import *  # NOQA
```

Comment:
I would avoid relative imports that go up the module path. These could be `from sentry.users.models ...`.